tests/testKinematicsSlice.cpp

```diff
@@ -81,6 +83,241 @@ TEST(Slice, InverseKinematics) {
   }
 }
 
+gtsam::Values jointVectorToValues(const Robot& robot,
+                                  const gtsam::Vector7& joints) {
```

Comment:
lol nice...

I have something similar in my cable robot `utils.py` file so we may consider in the future adding functions in `values.h` like `InsertJointAngles` (plural) and `JointAngles` (plural).